static/app/views/performance/cache/settings.ts

```diff
@@ -13,3 +14,9 @@ export const releaseLevelAsBadgeProps = {
 export const MODULE_TITLE = t('Cache');
 
 export const CHART_HEIGHT = 160;
+
+export const CACHE_BASE_URL = '/performance/cache';
+
+export const BASE_FILTERS: SpanMetricsQueryFilters = {
+  'span.op': 'cache.get_item', //  TODO - add more span ops as they become available, we can't use span.module because db.redis is also `cache`
+};
```

Comment:
You should talk to Will about this! If `span.module:cache` doesn't return what we expect, we should change what `span.module` does. You're making the module! You can help decide how the categorization works